jetty-websocket/websocket-common/src/main/java/org/eclipse/jetty/websocket/common/BlockingWriteCallback.java

```diff
@@ -37,7 +37,13 @@ public WriteBlocker acquireWriteBlocker() throws IOException
     {
         return new WriteBlocker(acquire());
     }
-    
+
+    protected long getIdleTimeout()
```

Comment:
You should annotate this with `@Override`. (I looked at the parent class to figure out what it actually does (override a default method from an interface?) but then I realized that, unlike in Scala, silent overrides are allowed in Java)